pages/spicedb/ops/performance.mdx

```diff
@@ -0,0 +1,90 @@
+import { Callout } from 'nextra/components'
+
+# Improving Performance
```

Comment:
note that this doc now has 3 sections: dispatch, materialize, namespace cache

@josephschorr any other sections worth adding?